### PR TITLE
HelpLine: Switching to text template

### PR DESCRIPTION
**Pull request by mintsoft**
###### Which issues (if any) does this fix?

Ref: https://github.com/duckduckgo/zeroclickinfo-goodies/issues/2767

###### People to notify (@mention interested parties)

---

Instant Answer Page: https://duck.co/ia/view/help_line

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @conorfl

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: t/HelpLine.t [hunk 1]
```diff
@@ -37,6 +38,19 @@ my @ok_queries = (
     'suicide silence',
 );
 
+sub build_test
+{
+    return test_zci(qr/24 Hour Suicide Hotline/, structured_answer => {
```
Comment (GuiltyDolphin) [inline t/HelpLine.t]: Wrap this with `re` please.

**Comment by GuiltyDolphin** [comment]
@mintsoft Merge conflicts! Apart from that, good to go!

**Comment by mintsoft** [comment]
Dammit I knew I shouldn't have merged that big PR of yours @GuiltyDolphin

**Comment by mintsoft** [comment]
@GuiltyDolphin Conflicts fixed

**Comment by GuiltyDolphin** [comment]
@mintsoft Thanks!

You know you wanted to though :wink:

**Pull request merged**